samples/secure-mcp-sse-client-server/secure-mcp-client/src/main/java/io/quarkiverse/langchain4j/sample/PoemResource.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkiverse.langchain4j.sample;
+
+import dev.langchain4j.agent.tool.Tool;
+import dev.langchain4j.service.UserMessage;
+import io.quarkiverse.langchain4j.RegisterAiService;
+import io.quarkiverse.langchain4j.mcp.runtime.McpToolBox;
+import io.quarkus.security.Authenticated;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+@Path("/poem")
+@Authenticated
+public class PoemResource {
+
+    static final String USER_MESSAGE = """
+            Write a short 1 paragraph poem about a Java programming language.
+            Please start by greeting the currently logged in user by name and asking to enjoy reading the poem.""";
+
+    @RegisterAiService
+    public interface GeminiPoemService {
+        @UserMessage(USER_MESSAGE)
+        @McpToolBox
```

Comment:
This is unnecessary and only bloats the code. When you don't specify a `value` for `@McpToolBox`, it just takes all the available mcp clients.

Reply:
I am pretty sure that's **not** the case anymore, right @mariofusco ?

Reply:
Ok it seems it is necessary (not that I agree with this change, but that's for another discussion)

Reply:
Yes, I confirm it is now necessary to add that annotation to use the configured MCP clients. This is in line with what the `@ToolBox` annotation does and also I don't see how you could say that you don't want that a service could use any MCP tool. Anyway, I'm open to discuss this, so please let me know if you have any proposal to improve this.